File: src/fr/servlets/VisitorsManagement.java
package fr.servlets;

import java.io.IOException;
import javax.servlet.ServletException;
import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import javax.servlet.http.HttpSession;
import fr.beans.PlayerBean;
import fr.utility.VisitorsManagementUtil;

@WebServlet("/visitorsManagement")
public class VisitorsManagement extends HttpServlet {
	private static final long serialVersionUID = 1L;
       
   
    public VisitorsManagement() {
        super();
     
    }
	
	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
		HttpSession session = request.getSession(false);
		PlayerBean player = (PlayerBean) session.getAttribute("user");

		if (session != null && player != null) {
			String statGV = request.getParameter("statusGV");
			String statDV = request.getParameter("statusDV");
			String statNV = request.getParameter("statusNV");

			/**Generer des visiteurs**/
			if ((statGV != null) && statGV.equals("okGV")) {
				VisitorsManagementUtil.generateVisitors(request, response);
			} /** Gerer la journée des visiteurs**/
			else if ((statDV != null) && statDV.equals("okDV")) {
				VisitorsManagementUtil.visitorsDay(request, response);
			} /** Depart des visiteurs la nuit**/
			else if ((statNV != null) && statNV.equals("okNV")) {	
				VisitorsManagementUtil.deleteVisitors(request, response);
			}
		}
	}

}

File: WebContent/assets/scripts/visitorsManagement.js

	function visitors() {
		let object ={};
		let hourT = null;
				hourT = sessionStorage.getItem("hour");
				
					//Generer des visiteurs a l'heure 1
					if(hourT == 1){
						var statusGV = "okGV";
						var callback = function(donnees) {
							$('#visitors').empty();
							$('#money').empty();
							$('#visitors').append("Visitors : "+ donnees.visitors);
							$('#money').append("Money : " + donnees.money + " Z");
						};
						
						object = {"statusGV" : statusGV};
						server.monAjax(object, "visitorsManagement", callback, 'POST');
						
					}
					//La journée des visiteurs
					 if(hourT > 1 && hourT <7){
						 var statusDV = "okDV";
						 var callback = function(donnees) {
							 $('#money').empty();
							$('#satisfaction').empty();
							$('#satisfaction').append("Satisfaction : " + donnees.satisfaction);
							$('#money').append("Money : " + donnees.money + " Z");
						 };
							
							object = {"statusDV" : statusDV};
							server.monAjax(object, "visitorsManagement", callback, 'POST');	 
					 }
					 
					 //Le Depart des visiteurs la nuit
					 if(hourT == 7){
						 var statusNV = "okNV";
							var callback = function(donnees) {
								$('#visitors').empty();
								$('#visitors').append("Visitors : "+ donnees.visitors);
							};
							
							object = {"statusNV" : statusNV};
							server.monAjax(object, "visitorsManagement", callback, 'POST');
					 }			
			}	